config/zones.json

```diff
@@ -1524,12 +1524,37 @@
     ]
   },
   "US-NY": {
+    "capacity": {
+      "coal": 1113,
+      "gas": 11686,
+      "hydro": 3538,
+      "hydro storage": 1297,
+      "nuclear": 5292,
+      "oil": 8260,
+      "solar": 250,
+      "wind": 1010,
+      "biomass": 50
+    },
     "parsers": {
       "production": "US_NY.fetch_production"
     },
     "flag_file_name": "us.png",
     "contributors": [
       "https://github.com/systemcatch"
```

Comment:
Missing comma at end of line.

Reply:
@corradio I thought that too, but every final line in Capacity like 1536 above ie 1516, 1480, etc. does not have a comma.  I may have messed everything up with my wrong PR requests up top....  "Merge branch 'master' into master 4061f24"

All things equal, I think best for you to close this PR (deny all changes), then I wipe/erase my fork & start fresh.  I can replicate any losses easily enough & I'll be able to see if I can get the mockerserver running from the start. 

Plus I have info ready for PJM, MISO & maybe SWPP.

Reply:
@HansHyde I'm talking about the first line.
This is invalid json:
```json
"contributors": [
    "https://github.com/systemcatch"
    "https://github.com/hanshyde"
```

However, this is valid:
```json
"contributors": [
    "https://github.com/systemcatch",
    "https://github.com/hanshyde"
```